test-data/unit/stubgen.test

```diff
@@ -355,12 +396,17 @@ __all__ = ['p']
 import urllib.parse as p
 
 [case testExportModuleInPackageUnsupported_import]
-import urllib.parse
 __all__ = ['urllib']
 [out]
 # Names in __all__ with no definition:
 #   urllib
 
+[case testExportModuleInPackage_import]
+import urllib.parse
+__all__ = ['urllib']
+[out]
+import urllib.parse
```

Comment:
I think there should still be an error comment in output like
```
# Names in __all__ with no definition:
#    urllib
```
before or after the import statement.